rest-api/src/test/java/io/quarkus/qe/SearchResourceTest.java

```diff
@@ -52,6 +52,20 @@ public void testRepositoriesQuery() {
         thenRepositoryIsFound();
     }
 
+    @Test
+    public void testRepositoriesQueryFilterByQuarkusVersion() {
+        givenExistingRepository();
+        whenRunGraphqlRepositoriesQuery(entity.quarkusVersion.id);
```

Comment:
Entity is a test field, add a new whenRunGraphqlRepositoriesQuery with zero arguments to improve readability. Actually, the version parameter should be optional.